src/tests/examples/code_generator.py

```diff
@@ -0,0 +1,110 @@
+# Code used to generate test functions
```

Comment:
question : where is this program called or used ?

Reply:
It must be called manually. The code is generated once and for all